### PR TITLE
i18n(zh-cn): Create `hashnode.mdx`

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Create `hashnode.mdx`

#### Related issues & labels (optional)

- #7374

### CODE REVIEW

File: src/content/docs/zh-cn/guides/cms/hashnode.mdx [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+## 使用 Astro 和 Hashnode 创建博客
+
+本指南使用 [`graphql-request`](https://github.com/jasonkuhrt/graphql-request)，一个与 Astro 配合良好的简约 GraphQL 客户端，将你的 Hashnode 数据引入你的 Astro 项目。
+
+### 前提条件
+
+1. 一个 Hashnode 博客
+2. 已安装了 [graphql-request](https://github.com/jasonkuhrt/graphql-request) 包的 Astro 项目。
+
+此示例将创建一个首页，列出帖子并链接到动态生成的各个帖子页面。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/hashnode.mdx:57]:
```suggestion
此示例将创建一个首页，列出所有文章并链接到动态生成的各个文章页面。
```

File: src/content/docs/zh-cn/guides/cms/hashnode.mdx [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+    `<Fragment />` 是 Astro 内置的一个组件，它可以帮助你避免使用不必要的包裹元素。当从 CMS（例如 Hashnode 或 [WordPress](/zh-cn/guides/cms/wordpress/)）获取 HTML 时，这一点尤其有用。
+    :::
+</Steps>
+
+### 发布你的网站
+要部署你的网站，请访问我们的[部署指南](/zh-cn/guides/deploy/)并按照你选择的托管提供商的指示操作。
+
+## 社区资源
+
+- [`astro-hashnode`](https://github.com/matthiesenxyz/astro-hashnode) 在 GitHub 上
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/hashnode.mdx:352]:
```suggestion
- GitHub 上的 [`astro-hashnode`](https://github.com/matthiesenxyz/astro-hashnode)
```

**Review by liruifengv — APPROVED** [review]
LGTM. Thanks!

**Review by dreyfus92 — APPROVED** [review]
Thank you both 😁

**Pull request merged**